src/section-multiplication-and-division-of-rational-expressions.mbx

```diff
@@ -43,15 +43,16 @@
 			<mrow>\frac{ax}{ay}\amp=\frac{x}{y}</mrow>
 			<mrow>\frac{a+x}{a+y}\amp\ne\frac{x}{y}</mrow>
 			<mrow>\frac{a-x}{a-y}\amp\ne\frac{-x}{-y}</mrow>
-		</md>This is not the first time a properties works for multiplication/division, but not for addition/subtraction. The same is true for square root:<md>
+		</md>This is not the first time a property works for multiplication/division, but not for addition/subtraction. The same is true for square root:<md>
 			<mrow>\sqrt{xy}\amp=\sqrt{x}\sqrt{y}</mrow>
 			<mrow>\sqrt{\frac{x}{y}}\amp=\frac{\sqrt{x}}{\sqrt{y}}</mrow>
 			<mrow>\sqrt{x+y}\amp\ne\sqrt{x}+\sqrt{y}</mrow>
+			<mrow>\sqrt{x-y}\amp\ne\sqrt{x}-\sqrt{y}</mrow>
 		</md>Let's look at a few examples.</p>
 
 		<example>
 			<statement>
-				<p>Simplify <m>\frac{4x^3y}{2x}</m>.</p>
+				<p>Simplify <m>\frac{4x^3y}{2x}</m></p>
```

Comment:
Curious why removing periods? When I put on my copy editor's hat, I'm going to make the punctuation and grammar meet standard rules, so I'd be putting the period back here.